device.yml

```diff
@@ -60,172 +60,100 @@ registers:
-  FrequencyDI1:
-    <<: *frequency
-    address: 48
-    description: Specifies the sound frequency to be played when triggering DI1
-  FrequencyDI2:
-    <<: *frequency
-    address: 49
-    description: Specifies the sound frequency to be played when triggering DI2
   AttenuationLeftDI0:
     <<: *attenuation
-    address: 50
-    description: Left channel's attenuation (1 LSB is 0.5dB) when triggering DI0
+    address: 45
+    description: Left channel's attenuation (1 LSB is 0.1dB) when triggering DI0
   AttenuationLeftDI1:
     <<: *attenuation
-    address: 51
-    description: Left channel's attenuation (1 LSB is 0.5dB) when triggering DI1
-  AttenuationLeftDI2:
-    <<: *attenuation
-    address: 52
-    description: Left channel's attenuation (1 LSB is 0.5dB) when triggering DI2
+    address: 46
+    description: Left channel's attenuation (1 LSB is 0.1dB) when triggering DI1
   AttenuationRightDI0:
     <<: *attenuation
-    address: 53
-    description: Right channel's attenuation (1 LSB is 0.5dB) when triggering DI0
+    address: 47
+    description: Right channel's attenuation (1 LSB is 0.1dB) when triggering DI0
   AttenuationRightDI1:
     <<: *attenuation
-    address: 54
-    description: Right channel's attenuation (1 LSB is 0.5dB) when triggering DI1
-  AttenuationRightDI2:
-    <<: *attenuation
-    address: 55
-    description: Right channel's attenuation (1 LSB is 0.5dB) when triggering DI2
-  AttenuationAndSoundIndexDI0:
-    <<: *attenuation
-    address: 56
-    length: 3
-    description: Sound index and attenuation to be played when triggering DI0 [Att R] [Att L] [Index]
-  AttenuationAndSoundIndexDI1:
-    <<: *attenuation
-    address: 57
-    length: 3
-    description: Sound index and attenuation to be played when triggering DI1 [Att R] [Att L] [Index]
-  AttenuationAndSoundIndexDI2:
-    <<: *attenuation
-    address: 58
-    length: 3
-    description: Sound index and attenuation to be played when triggering DI2 [Att R] [Att L] [Index]
-  AttenuationAndFrequencyDI0:
-    <<: *attenuation
-    address: 59
-    length: 2
-    description: Sound index and attenuation to be played when triggering DI0 [Att BOTH] [Frequency]
-  AttenuationAndFrequencyDI1:
-    <<: *attenuation
-    address: 60
-    length: 2
-    description: Sound index and attenuation to be played when triggering DI1 [Att BOTH] [Frequency]
-  AttenuationAndFrequencyDI2:
-    <<: *attenuation
-    address: 61
-    length: 2
-    description: Sound index and attenuation to be played when triggering DI2 [Att BOTH] [Frequency]
+    address: 48
+    description: Right channel's attenuation (1 LSB is 0.1dB) when triggering DI1
   Reserved2:
     <<: *reserved
-    address: 62
+    address: 49
   Reserved3:
     <<: *reserved
-    address: 63
+    address: 50
   Reserved4:
     <<: *reserved
-    address: 64
+    address: 51
   ConfigureDO0: &configureDO
-    address: 65
+    address: 52
     type: U8
     access: Write
+    default: 1
```

Comment:
```suggestion
    defaultValue: 1
```

Reply:
@glopesdev I am not able to compile using a defaultValue with an enumerator. Is there something I am missing? The types should be able to be correctly inferred but maybe an edge case in the current generator.